packages/ui/src/components/tabs.tsx

```diff
@@ -61,12 +68,24 @@ export function Tabs({
   items = [],
   persist = false,
   defaultIndex = 0,
+  updateAnchor = false,
   ...props
 }: TabsProps): React.ReactElement {
   const values = useMemo(() => items.map((item) => toValue(item)), [items]);
   const [value, setValue] = useState(values[defaultIndex]);
   const valuesRef = useRef(values);
   valuesRef.current = values;
+  const valueToIdMap = useMemo(() => {
+    const map = new Map<string, string>();
+    Children.forEach(props.children, (child, index) => {
```

Comment:
Thanks @fuma-nama - I've refactored the code to use ref/context instead of the Children API.

As an aside, do you think `ValueChangeContext` and `updateAnchor` should be renamed?

Reply:
I would prefer to it rename to `TabsContext`

Reply:
Yep makes sense - renamed in 8845f523